db/db.go

```diff
@@ -0,0 +1,29 @@
+package db
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+
+	"github.com/iterableio/api/config"
+)
+
+var DB *sqlx.DB
+
+func ConnectSQL() error {
```

Comment:
so whats the advantage of not having this shit be inside `init`? I feel like this can be cleaner, as the connection to the db will be self contained and the other packages dont have to worry about whether its connected or not, they will know for sure that it is